.github/workflows/before_merge.yaml

```diff
@@ -119,8 +119,6 @@ jobs:
           rm -rf tests/autogen_op/ # Remove old tests
           python3 tools/generate_input_variation_test_from_models.py
           python3 tools/generate_input_variation_test_from_models.py --merge=True
-          echo "Remove these files because it's heavy for running"
-          rm -f tests/autogen_op/Stable_Diffusion_V2/test_Stable_Diffusion_V2_aten_convolution_default.py
```

Comment:
Maybe this PR can just merged this two line, then the next time run workflow with `All` then the `test_Stable_Diffusion_V2_aten_convolution_default.py` will appear

Reply:
Sounds ok to me

Reply:
ok, I've rebased this PR and only remain this two line's change